doc/content/verification_and_validation/ver-1ja.md

```diff
@@ -0,0 +1,66 @@
+# ver-1ja
+
+Two problems (ver-1ja and [ver-1jb](ver-1jb.md)) demonstrate tritium decay, though any other isotope could have been chosen.
+The first (ver-1ja) is simple decay of mobile species in a slab.
+The second ([ver-1jb](ver-1jb.md)) is decay of trapped atoms in a similar slab but with a distributed trap concentration.
+This page presents ver-1ja.
+
+# Radioactive Decay of Mobile Tritium in a Slab
+
+## General Case Description
+
```

Comment:
```suggestion

!style halign=left
```

This adds some space between the heading and body text, for more pleasurable reading.